flood_adapt/object_model/hazard/physical_projection.py

```diff
@@ -6,14 +6,14 @@
 
 
 class PhysicalProjectionModel(BaseModel):
-    sea_level_rise: Optional[UnitfulLength] = 0
-    subsidence: Optional[UnitfulLength] = 0
-    rainfall_increase: Optional[float] = 0
-    storm_frequency_increase: Optional[float] = 0
+    sea_level_rise: Optional[UnitfulLength]  # TODO add default zero
```

Comment:
I think it works like this, but not sure. I did this a while ago but then deleted it in a code revision :-D so you would need to test this
```suggestion
    sea_level_rise: Optional[UnitfulLength]  = UnitfulLength(value=0., units="meters")
```